flixel/FlxCamera.hx

```diff
@@ -103,19 +103,19 @@ class FlxCamera extends FlxBasic
 	/**
 	 * Lower bound of the cameras scroll on the x axis
 	 */
-	public var minScrollX:Null<Float>;
+	public var minScrollX:Float = FlxMath.MIN_VALUE_FLOAT;
```

Comment:
I don't think we can change these types for now, that would be a fairly big breaking change.